src/javascript/app/pages/callback/callback.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Callback } from '@deriv-com/auth-client';
+
+import Cookies from 'js-cookie';
+import moment from 'moment';
+import Client from '../../base/client';
+import BinarySocket from '../../base/socket';
+import GTM from '../../../_common/base/gtm';
+import { get as getLanguage, urlLang } from '../../../_common/language';
+import { isStorageSupported, removeCookies } from '../../../_common/storage';
+import { urlFor } from '../../../_common/url';
+import { getPropertyValue } from '../../../_common/utility';
+import { getElementById } from '../../../_common/common_functions';
+
+const map_names = {
+    country             : 'residence',
+    landing_company_name: 'landing_company_shortcode',
+};
+
```

Comment:
For this component, I just took most of the authentication logic from `logged_in.js` and migrate it to the `Callback` component, which will be called when we receive the tokens from OIDC at `onSignInSuccess`